nrgpy/utils/utilities.py

```diff
@@ -88,20 +87,14 @@ def date_check(start_date, end_date, string):
         string including date to check
     """
     # LOGR & SymphonieClassic
-    if string.endswith(("dat", "rwd")):
-        date_format = "([0-9]{4}[0-9]{2}[0-9]{2})"
-        strp_format = "%Y%m%d"
+    date_format_no_underscore = "([0-9]{4}[0-9]{2}[0-9]{2})"
+    strp_format_no_underscore = "%Y%m%d"
     # ZX datafile
-    elif "@Y20" in string:
-        date_format = "(Y[0-9]{4}_M[0-9]{2}_D[0-9]{2})"
-        strp_format = "Y%Y_M%m_D%d"
+    date_format_zx = "(Y[0-9]{4}_M[0-9]{2}_D[0-9]{2})"
+    strp_format_zx = "Y%Y_M%m_D%d"
     # SymphoniePRO
-    elif string.lower().endswith("rld") or "_" in string:
-        date_format = "([0-9]{4}\-[0-9]{2}\-[0-9]{2})"
-        strp_format = "%Y-%m-%d"
-    else:
-        logger.error(f"{string} is an unknown file format")
-        print(f"{string} is an unknown file format")
+    date_format_with_underscore = "([0-9]{4}\-[0-9]{2}\-[0-9]{2})"
```

Comment:
please don't hate me... but that's a dash (or hyphen?), not an underscore. it's ok, but the variable name threw me off when I was expecting underscores...